core/views.py

```diff
@@ -8,9 +8,4 @@
 
 @app.route('/')
 def home():
-    try:
```

Comment:
I understand but if you want share the url of the main page you can't go there and copy it. Weird no ?